src/phoenix/server/main.py

```diff
@@ -367,6 +376,25 @@ def _get_pid_file() -> Path:
         scaffold_datasets=scaffold_datasets,
         phoenix_url=root_path,
     )
+    email_sender = None
+    if mail_sever := get_env_smtp_hostname():
+        assert (mail_username := get_env_smtp_username()), "SMTP username is required"
+        assert (mail_password := get_env_smtp_password()), "SMTP password is required"
+        assert (mail_from := get_env_smtp_mail_from()), "SMTP mail_from is required"
```

Comment:
Can we add a default mail from so this env var is not required to be passed by the user? We don't even necessarily need to make this configurable on a first pass.

Reply:
ok, but let's punt on that for now

Reply:
Okay to merge the PR, but I think we should add this default before launch.